aspnetcore-webapi/DotNetURLShortener/DotNetURLShortener/Services/UrlShortenerService.cs

```diff
@@ -0,0 +1,55 @@
+﻿using DotNetURLShortener.Contracts;
+
+namespace DotNetURLShortener.Services;
+
+public class UrlShortenerService : IUrlShortenerService
+{
+    private const string AlphaNumericChars = "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
+    private const int ShortCodeLength = 5;
+
+    public Dictionary<string, string> UrlDict { get; set; } = [];
+
+    public string GetShortCode(string longUrl)
+    {
+        static string GenerateShortCode()
+        {
+            var shortCode = new char[ShortCodeLength];
+            for (var i = 0; i < ShortCodeLength; i++)
+            {
+                shortCode[i] = AlphaNumericChars[Random.Shared.Next(AlphaNumericChars.Length)];
+            };
+
+            return new string(shortCode);
```

Comment:
You can use [Random.Shared.GetItems()](https://learn.microsoft.com/en-us/dotnet/api/system.random.getitems?view=net-8.0#system-random-getitems-1(system-readonlyspan((-0))-system-span((-0)))) here in combination with [string.Create](https://learn.microsoft.com/en-us/dotnet/api/system.string.create?view=net-8.0#system-string-create-1(system-int32-0-system-buffers-spanaction((system-char-0))))

Reply:
@cjaynjoku this is still pending

Reply:
Hello @elfico, I have changed the implementation of that method to include the methods @jshergal suggested here.

Reply:
Thanks. Somehow my updated code wasn't reflecting. I had to pull again.
Sorry for the bother